test_scripts/Defects/8_0/3788/001_allow_RC_permission_in_case_App_is_registered_with_MEDIA_appHMIType.lua

```diff
@@ -0,0 +1,60 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3788
+---------------------------------------------------------------------------------------------------
+-- Description: Check SDL assigned RC permissions for the App in case:
+-- - 'REMOTE_CONTROL' appHMIType is assigned for App by policy
+-- - app is registered with 'MEDIA' appHMIType
```

Comment:
I would propose to rewrite description in a following way:
> Check SDL assigns RC permissions for the App in case:
> - Policy table contains an information about app with 'REMOTE_CONTROL' appHMIType
> - App is registered with 'MEDIA' appHMIType